lita_config.rb

```diff
@@ -1,14 +1,5 @@
 require 'bundler'
-
-module Lita
-  def self.env
-    ENV["LITA_ENV"] || :development
-  end
-
-  def self.env?(env=:development)
-    self.env.to_s == env.to_s
-  end
-end
+require_relative 'lita-zooniverse/lib/lita-zooniverse'
```

Comment:
Wouldn't this mean all of those gem deps (httparty etc) get required before we lock versions using bundler?